### ISSUE TITLE
Cannot Choose Box-shadow Position For Image Widget

### ISSUE THREAD

**Issue by MarkiusFlavious** [elementor/elementor#22039]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The option to set box-shadow position for the image widget is hidden below the page in the editor.

### Steps to reproduce

1) Go to the Elementor Editor for anything.
2) Add an image widget.
3) Go to the styles tab for the image widget.
4) Scroll down and click on the pencil next to Box Shadow.
5) Only the options for Colour, Horizontal, Vertical, Blur and Spread are visible.
6) The option to choose Position is hidden below the screen and cannot be selected.
7) This is a major issue because I had to bake shadows into my images to style them correctly. 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.4.28
	PHP Version: 8.0.28
	PHP Memory Limit: 96M
	PHP Max Input Vars: 2500
	PHP Max Post Size: 32M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://mark.digitalhumanitydev.co.za
	Home URL: https://mark.digitalhumanitydev.co.za
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 40M
	Max Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: en-ZA
	Timezone: Africa/Johannesburg
	Admin Email: developeradmin@digitalhumanity.co.za
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_ZA
	User Agent: Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/112.0

== Active Plugins ==
	Elementor
		Version: 3.12.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.2
		Author: Elementor.com

== Elements Usage ==
	
	archive : 1
		container : 1
		jet-listing-grid : 1
	header : 2
		container : 8
		image : 1
		jet-ajax-search : 1
		jet-nav-menu : 1
		mega-menu : 1
		nav-menu : 1
		theme-site-logo : 2
	wp-post : 0
		button : 1
		container : 2
		icon : 1
		image : 1
		jet-listing-dynamic-field : 1
		jet-listing-dynamic-image : 1
		theme-post-title : 1
	single-post : 1
		container : 3
		gallery : 1
		jet-listing-dynamic-field : 5
		theme-post-featured-image : 1
		theme-post-title : 1
	single-page : 1
		container : 5
		form : 1
		heading : 2
		jet-ajax-search : 1
		jet-listing-grid : 1
		jet-smart-filters-active : 1
		loop-grid : 1
	loop-item : 1
		container : 1
		heading : 1
		jet-listing-dynamic-field : 3
		jet-listing-dynamic-image : 1
		jet-listing-dynamic-link : 1
	jet-listing-items : 0
		container : 1
		heading : 1
		jet-listing-dynamic-field : 3
		jet-listing-dynamic-image : 1
		jet-listing-dynamic-link : 1

== Settings ==
	
	cpt_support: post, page, jet-menu, jet-woo-builder, e-landing-page, product, jet-engine, properties
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: fallback

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimised DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header &amp; Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Active
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active
	Scroll Snap: Active by default
	Menu: Active

== Log ==
	
JS: showing 4 of 4JS: 2022-11-11 07:06:19 [error X 1][https://master.digitalhumanitydev.co.za/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1:4:51824] Cannot read properties of undefined (reading \&#039;replace\&#039;) 
JS: 2023-04-03 13:51:56 [error X 4][https://mark.digitalhumanitydev.co.za/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.1:3:785745] Cannot read properties of undefined (reading &#039;localized&#039;) 
JS: 2023-04-07 18:55:19 [error X 1][https://mark.digitalhumanitydev.co.za/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.1:3:893419] elementorFrontend is not defined 
JS: 2023-04-12 14:25:44 [error X 1][https://mark.digitalhumanitydev.co.za/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: &quot;view34024&quot;) has already been destroyed and cannot be used. 

Log: showing 20 of 322023-03-08 08:30:29 [info] elementor-pro::elementor_pro_updater Started 
2023-03-08 08:30:29 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-08 08:30:29 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-08 08:30:29 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.2',
  'to' => '3.11.4',
)]
2023-04-03 14:16:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.3',
  'to' => '3.12.1',
)]
2023-04-03 14:16:14 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.4',
  'to' => '3.12.1',
)]
2023-04-03 14:16:15 [info] elementor-pro::elementor_pro_updater Started 
2023-04-03 14:16:15 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-03 14:16:15 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-03 14:16:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.4',
  'to' => '3.12.1',
)]
2023-04-03 14:16:15 [info] elementor::elementor_updater Started 
2023-04-03 14:16:15 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-03 14:16:15 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-03 14:16:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.3',
  'to' => '3.12.1',
)]
2023-04-03 14:16:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.4',
  'to' => '3.12.1',
)]
2023-04-03 14:16:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.3',
  'to' => '3.12.1',
)]
2023-04-12 16:32:09 [info] elementor-pro::elementor_pro_updater Started 
2023-04-12 16:32:09 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-12 16:32:09 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-12 16:32:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.1',
  'to' => '3.12.2',
)]

PHP: showing 7 of 7PHP: 2023-01-31 03:05:11 [notice X 5][/usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/plugin.php(154): class type include()array (
  0 => '/usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/plugin.php(154): class type include()array (
  0 => '/usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]
PHP: 2023-03-08 08:29:04 [notice X 51][/usr/www/users/mastekgnpw/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-03 14:21:43 [warning X 28][/usr/www/users/markdnewdn/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key &quot;data&quot; [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-03 14:38:00 [warning X 3][/usr/www/users/markdnewdn/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-03 23:08:38 [error X 6][/usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager-&gt;ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /usr/www/users/markdnewdn/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /usr/www/users/markdnewdn/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module-&gt;handle_ajax_request(&#039;&#039;)
#4 /usr/www/users/markdnewdn/wp-includes/class-wp-hook.php(332): WP_Hook-&gt;apply_filters(&#039;&#039;, Array)
#5 /usr/www/users/markdnewdn/wp-includes/plugin.php(517): WP_Hook-&gt;do_action(Array)
#6 /usr/www/users/markdnewdn/wp-admin/admin-ajax.php(188): do_action(&#039;wp_ajax_element...&#039;)
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-12 16:12:10 [warning X 2][/usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: /usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /usr/www/users/markdnewdn/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-04-12 16:13:30 [warning X 1][/usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key &quot;condition_type&quot; [array (
  'trace' => '
#0: /usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /usr/www/users/markdnewdn/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(84): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /usr/www/users/markdnewdn/wp-content/plugins/elementor/data/base/endpoint.php(166): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> update_item()
#3: /usr/www/users/markdnewdn/wp-content/plugins/elementor/data/base/endpoint.php(307): Elementor\Data\Base\Endpoint -> base_callback()
#4: /usr/www/users/markdnewdn/wp-includes/rest-api/class-wp-rest-server.php(1181): Elementor\Data\Base\Endpoint -> Elementor\Data\Base\{closure}()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==

**Comment by MarkiusFlavious** [comment]
After playing around with custom CSS, I realize that this is less like a bug report and more like a feature request.
